todolist/app.go

```diff
@@ -25,6 +25,10 @@ func (a *App) AddTodo(input string) {
 	a.Load()
 	parser := &Parser{}
 	todo := parser.ParseNewTodo(input)
+	if todo == nil {
+		fmt.Println("What to do?")
```

Comment:
How about something like "I need more information.  Try something like 'todo a chat with @bob due tom'"

That way we give the user a bit more context on how the functionality works.